### PR TITLE
Style for the icon in the modal header

### PR DESCRIPTION
**Pull request by zoeesilcock**
modal-header-icon

### CODE REVIEW

**Comment by eliashedlund** [comment]
LGTM :+1:

File: src/mnd-bootstrap/javascript/_modal.scss [hunk 1]
```diff
@@ -71,6 +71,11 @@ category: JavaScript
   a {
     text-decoration: underline;
   }
+
+  .header-icon {
```
Comment (jorkas) [inline src/mnd-bootstrap/javascript/_modal.scss]: Maybe we should use a smaller one for smaller screens and use this one only for larger screens?

**Comment by captureteam** [comment]
We need it (for the demo) so we merge it. Lets make it responsive later!

**Comment by jorkas** [comment]
@captureteam `mobile first`!

**Comment by captureteam** [comment]
:+1:

**Pull request merged**